frontend/src/router.tsx

```diff
@@ -0,0 +1,26 @@
+import {
+    BrowserRouter,
+    Routes,
+    Route,
+  } from "react-router-dom";
+import About from "./pages/About";
+
+import Home from "./pages/Home";
+import CanOfferShelter from "./pages/Signup/CanOfferShelter";
+import NeedShelter from "./pages/Signup/NeedShelter";
+
+export default function Router() {
+    return (
+        <BrowserRouter>
+            <Routes>
+                {/* Unprotexted routes */}
```

Comment:
I think you mean Unprotected

Reply:
Yeah, of course, forgot to turn on spellcheck